format/base.go

```diff
@@ -105,28 +105,28 @@ func gfmAccordion(title, body string) string {
 // following:
 //
 //	accordion := gfmAccordionHeader("Accordion Title") + "Accordion Body" + gfmAccordionTerminator()
-func gfmAccordionHeader(title string) string {
+func GfmAccordionHeader(title string) string {
 	return fmt.Sprintf("<details><summary>%s</summary>\n<p>\n\n", title)
 }
 
 // gfmAccordionTerminator generates the code necessary to terminate an
 // accordion after the body. It is expected to be used in conjunction with
 // gfmAccordionHeader(). See gfmAccordionHeader for a full description.
-func gfmAccordionTerminator() string {
+func GfmAccordionTerminator() string {
 	return "</p>\n</details>\n\n"
 }
 
 // paragraph formats a paragraph with the provided text as the contents.
-func paragraph(text string) string {
-	return fmt.Sprintf("%s\n\n", escape(text))
+func Paragraph(text string) string {
+	return fmt.Sprintf("%s\n\n", Escape(text))
 }
 
 var (
 	specialCharacterRegex = regexp.MustCompile("([\\\\`*_{}\\[\\]()<>#+-.!~])")
 	urlRegex              = xurls.Strict() // Require a scheme in URLs
 )
 
-func escape(text string) string {
+func Escape(text string) string {
```

Comment:
I expect the linter will error out telling you this needs to have a documentation comment